packages/trace-viewer/src/ui/snapshotTab.tsx

```diff
@@ -110,6 +110,9 @@ export const SnapshotView: React.FunctionComponent<{
 
   React.useEffect(() => {
     (async () => {
+      if (snapshotInfo.snapshotInfoUrl === snapshotUrls?.snapshotInfoUrl)
```

Comment:
This effect only depends on the `snapshotUrls` instance, so Reacty fix would be on the call site. I can see that 

```js
  const snapshotUrls = React.useMemo(() => {
    const snapshot = snapshots[snapshotTab];
    return snapshot ? extendSnapshot(snapshot, shouldPopulateCanvasFromScreenshot) : undefined;
  }, [snapshots, snapshotTab, shouldPopulateCanvasFromScreenshot]);
```

so the `snapshots` array is new, which is because `action` is new, etc. If we want to be reacty, we should make sure that those objects "stay the same" in the live mode, that way all the little refreshes go away, not just the snapshot one.